README.md

```diff
@@ -0,0 +1,58 @@
+# BIOL:4386 Course Project - Reproducing *Deriving genotypes from RAD-seq short-read data using Stacks*
+## Project Homework 1 - Feb 12 2023
+
+## Reference 
+Rochette, N., Catchen, J. Deriving genotypes from RAD-seq short-read data using Stacks. Nat Protoc 12, 2640–2659 (2017). https://doi.org/10.1038/nprot.2017.123
+
+## Introduction 
+Use of restriction-site associated DNA sequencing (RAD seq) has increasingly been utilized to genotype populations without non-model organisms as a tool for population genetics analysis. To produce a reliable set of loci for downstream analysis, understanding of bioinformatic programs such as STACKS is necessary. 
+
+STACKS is a software pipeline for building loci from short-read sequences, such as those generated on the Illumina sequencing results. STACKS was specifically built to work with restriction-enzyme based data, such as RAD-seq, in order to build genetic maps and conduct population genomics and phylogeography. 
+
+The pipeline of STACKS begins with the denovo_map.pl program that executes the STACKS pipeline by running each of the following STACKS components individually: 
```

Comment:
these are too much into the weeds. how can you explain the principle of this program without getting into this level of details? you, the person doing the work, certainly needs to understand all of them. but most likely the reader just want to know 1) why a specialized program is needed vs the general purpose workflow, e.g., mapping reads to the genotype and assemble the loci; 2) how it works, focusing on the modifications or unique steps compared with the general workflow